src/quo2/screens/network_amount.cljs

```diff
@@ -0,0 +1,70 @@
+(ns quo2.screens.network-amount
+  (:require [quo.previews.preview :as preview]
+            [quo.react-native :as rn]
+            [quo2.components.network-amount :as network-amount]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]))
+
+(def ^:private networks
+  [#:network{:icon :main-icons2/ethereum
```

Comment:
why `#:network` is needed ?

Reply:
I used namespaced keys to clarify it's a description of a network. Since it's only used as dummy data on this example screen it has no other benefits, so I will remove it.

Reply:
Changed.